src/dojos/cd-find-dojo.vue

```diff
@@ -37,7 +37,25 @@
           <no-results-mobile @toggleMap="toggleMap" class="cd-find-dojo__no-results-message--mobile visible-xs"></no-results-mobile>
         </h4>
       </div>
-      <dojo-map :center="coordinates" :class="{ 'cd-find-dojo__results-map': true, 'cd-find-dojo__results-map--hidden': !showMap }" :dojos="allActiveDojos"></dojo-map>
+      <div class="cd-find-dojo__right-column">
+        <dojo-map :center="coordinates" :class="{ 'cd-find-dojo__results-map': true, 'cd-find-dojo__results-map--hidden': !showMap }" :dojos="allActiveDojos"></dojo-map>
```

Comment:
class will always be "cd-find-dojo__results-map" ?

Reply:
I'm not sure why it's like that, @DanielBrierton do you know what this is?

Reply:
Because it uses Vue's special class binding: https://vuejs.org/v2/guide/class-and-style.html

Basically we want the class to be `cd-find-dojo__results-map` when `showMap` is `true`, and `cd-find-dojo__results-map cd-find-dojo__results-map--hidden` when `showMap` is `false`

Reply:
okay cool so we leave it the way it is

Reply:
I'd better use class & v-bind:class
It's clearer imho what's "always here" compared to what's dynamic
```<div class="static"
     v-bind:class="{ active: isActive, 'text-danger': hasError }">
</div>```
Thoughts @DanielBrierton

Reply:
I did not cop that this was possible... Yes, this is much nicer.

```
class="cd-find-dojo__results-map :class="{'cd-find-dojo__results-map--hidden': !showMap }"
```